sql/moz-fx-data-shared-prod/adjust/adjust_derived/view.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE VIEW
+    `moz-fx-data-shared-prod.analysis.adjust_derived_v1`
```

Comment:
is `analysis` the right dataset?  May also want to confirm the table name?

Reply:
yes! the dataset will be adjust_derived. I wanted to make sure my logic was right before changing to the proper name